feature-libs/organization/administration/components/user/details/user-details.component.html

```diff
@@ -38,14 +38,31 @@
     <div class="property">
       <label>{{ 'orgUser.roles' | cxTranslate }}</label>
       <ul class="value">
-        <li
-          *ngFor="let role of model.roles; let isLast = last"
-          [innerText]="'organization.userRoles.' + role | cxTranslate"
-        ></li>
+        <ng-container *ngFor="let role of model.roles">
+          <li
+            *ngIf="availableRoles.includes(role)"
+            [innerText]="'organization.userRoles.' + role | cxTranslate"
+          ></li>
+        </ng-container>
         <li *ngIf="model.roles?.length === 0">-</li>
       </ul>
     </div>
 
+    <ng-container *ngIf="hasRight(model)">
+      <div class="property">
+        <label>{{ 'orgUser.rights' | cxTranslate }}</label>
+        <ul class="value">
+          <ng-container *ngFor="let role of model.roles; let isLast = last">
```

Comment:
Seems as if the last element is not used in this template

Reply:
Yes can be removed, also appears in another file without usage, can be removed there too.